src/tictactoe_clojure/board.clj

```diff
@@ -1,28 +1,82 @@
-(ns tictactoe-clojure.board)
+(ns tictactoe-clojure.board
+  (:require [tictactoe-clojure.console :as console])
+  (:require [clojure.math.numeric-tower :as math]))
 
-(def new-board [0 1 2 3 4 5 6 7 8])
+(defn get-size
```

Comment:
Initially, I felt like this was the board's responsibility because it is setting the size of the board. But since the board is not even relying on `get-size` function, it probably makes more sense to have this under the game-setup name space.